File: Javascript/level 2/Find a decimal number.js
function numberOfPrime(n) {
	var result = 0;
	// 함수를 완성하세요.
	var cnt=0;
	for(var a=2;a<=n;a++){
    cnt=0;
  	for(var b=1;b<=a;b++){
    		if(a%b==0)
          cnt++;
    }
    if(cnt==2)
      result++;
  }
	return result;
}


// 아래는 테스트로 출력해 보기 위한 코드입니다.
console.log( numberOfPrime(10) )

File: CPP/level 2/Find a decimal number.cpp
#include<iostream>
using namespace std;

int numOfPrime(int n)
{
	int answer = 0;
  int cnt=0;
	for(int a=2;a<=n;a++){
    cnt=0;
  	for(int b=1;b<=a;b++){
    		if(a%b==0)
          cnt++;
    }
    if(cnt==2)
      answer++;
  }
	return answer;
}

int main()
{
	int testCase = 10;
	int testAnswer = numOfPrime(testCase);

	cout<<testAnswer;
}
